_docs_integrate/upgrade-existing-online-accounts-of-my-users.md

```diff
@@ -0,0 +1,22 @@
+---
+# Start automatic generation
+permalink: integrate/upgrade-existing-online-accounts-of-my-users
+published: false
+title: "upgrade existing online accounts of my users"
+type: scenario
+toc: true
+properties:
+  - id: SC108
+  - category: Get in touch with other Identities
+  - description: Verein/Versicherung mit Identity Provider hat einen bestehenden Kunden mit Username/Password Nutzer kommt auf Website Nutzer muss sich anmelden personalisiertes und sessionbasiertes Template erzeugen Nutzer scannt QR Code ein Nutzer stellt Kontaktanfrage Kontaktanfrage wird mit session und Nutzer gemappt Nutzer wird auf enmeshed gehoben (z.B. adresse als Attribut speichern) Refresh der Website mittels Websockets antriggern Bestehender Nutzer kann enmeshed benutzen
```

Comment:
This is more of a side note: I have noticed that the 5 Connector scenarios SC108, SC109, SC110, SC111 and SC112, which have been newly added to the documentation, have a 'description' written in German instead of English. To be consistent, this should be changed to English. (Here and in Excel)

The same applies to the corresponding 5 App scenarios SC014, SC015, SC016, SC017 and SC018.

Reply:
i would leave the description in german for the time being to make it easier to describe unclear scenarios. This field is not displayed publicly